src/elements/content-sidebar/activity-feed/annotation-thread/errors.js

```diff
@@ -0,0 +1,14 @@
+import {
+    ERROR_CODE_DELETE_ANNOTATION,
+    ERROR_CODE_EDIT_ANNOTATION,
+    ERROR_CODE_FETCH_ANNOTATION,
+} from '../../../../constants';
+import messages from './messages';
+
+const AnnotationErrors = {
+    [ERROR_CODE_FETCH_ANNOTATION]: messages.errorFetchAnnotation,
+    [ERROR_CODE_EDIT_ANNOTATION]: messages.errorEditAnnotation,
+    [ERROR_CODE_DELETE_ANNOTATION]: messages.errorDeleteAnnotation,
+};
+
+export default AnnotationErrors;
```

Comment:
Can we use camelCase (`annotationErrors`) since this is a regular variable rather than a component?